samples/sample-resource-deployment/package.nls.json

```diff
@@ -1,5 +1,5 @@
 {
-	"extension-displayName": "Sample resource deployment extension for Azure Data Studio",
+	"extension-displayName": "Sample resource deployment extension for  Data Studio",
```

Comment:
Why did you delete Azure?

Reply:
fixed.

Reply:
Adding back. It was inadvertent. Adding back.